### PR TITLE
Feature http eventlog

### PR DESCRIPTION
**Pull request by Jan-M**

### CODE REVIEW

File: src/main/java/de/zalando/zmon/dataservice/data/HttpEventLogger.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        }
+
+        try {
+            Request request = Request.Post(forwardUrl + "/")
+                    .bodyString(mapper.writeValueAsString(new HttpEvent(new Date(), type, values)), ContentType.APPLICATION_JSON);
+
+            async.execute(request, new FutureCallback<Content>() {
+
+                public void failed(final Exception ex) {
+                    metrics.markProxyError();
```
Comment (hjacobs) [inline src/main/java/de/zalando/zmon/dataservice/data/HttpEventLogger.java]: markEventLogError

File: src/main/java/de/zalando/zmon/dataservice/data/HttpEventLogger.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        }
+    }
+
+    public void log(EventType type, Object... values) {
+        if(!enabled) {
+            return;
+        }
+
+        try {
+            Request request = Request.Post(forwardUrl + "/")
```
Comment (hjacobs) [inline src/main/java/de/zalando/zmon/dataservice/data/HttpEventLogger.java]: forwardUrl?

**Comment by hjacobs** [comment]
:+1:

**Comment by Jan-M** [comment]
:+1:

**Pull request merged**